projects/packages/videopress/src/client/block-editor/hooks/use-video-data-update/index.ts

```diff
@@ -192,7 +192,10 @@ export function useSyncMedia(
 			return;
 		}
 
-		if ( ! videoData || Object.keys( videoData ).length === 0 ) {
+		if (
+			! videoData ||
+			Object.keys( videoData ).filter( key => key !== 'private_enabled_for_site' ).length === 0
```

Comment:
We should probably update this condition using the [videoFieldsToUpdate](https://github.com/Automattic/jetpack/blob/8f80eee187504064b20e5c3b5012a28cb2008fc8/projects/packages/videopress/src/client/block-editor/hooks/use-video-data-update/index.ts#L66-L75) array.
Not 100% sure, and don't take it as a blocker.

Reply:
Will leave that for a follow-up :+1: